guide/ops/locations/index.md

```diff
@@ -62,8 +62,12 @@ brooklyn.location.jclouds.aws-ec2.credential=s3cr3tsq1rr3ls3cr3tsq1rr3ls3cr3tsq1
 
 And in this case you can reference the location in YAML with `location: jclouds:aws-ec2`.
 
-The Getting Started [template brooklyn.properties]({{ site.path.guide }}/start/brooklyn.properties) contains more examples 
-of configuring cloud endpoints, including information on credential types used in different clouds.
+Brooklyn irons out many of the differences between clouds so that blueprints run similarly
+in a wide range of locations, including setting up access and configuring images and machine specs.
+The configuration options are described in more detail below.
+
+In some cases, a cloud provider may offer special features or unexpected quirks;
+known issues are described a the **[More Details for Specific Clouds](more-clouds.html)** page.
```

Comment:
How about:

In some cases, a cloud provider may offer special features or have unexpected quirks. Known issues such as these are described in **[More Details for Specific Clouds](more-clouds.html)**.